packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1537,16 +1530,15 @@ export const audiusBackend = ({
     }
   }
 
-  async function markAllNotificationAsViewed() {
+  async function markAllNotificationAsViewed({ userId }: { userId: ID }) {
```

Comment:
Note for myself: I think this pattern might be leftover from my initial attempt to completely remove current user state from libs and I need to go back through this file and switch it back to reading the current user ID from libs state.